.clang-format

```diff
@@ -30,7 +30,7 @@ DerivePointerAlignment: false
 DisableFormat: false
 ExperimentalAutoDetectBinPacking: true
 IndentCaseLabels: true
-IndentWidth: 2
+IndentWidth: 4
```

Comment:
Also for the moose we use double indent for continuation indents

```cpp
void functionWithALongName(
        int some_variable_with_a_really_long_name_that_requires_wrapping) {
    //Do stuff
}
```

Obviously avoid stupid long names, but I can't come up with a contrived example off the top of my head right now.